repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml

```diff
@@ -187,10 +188,19 @@
                             <hippo:configVariant variant="published" availabilities=""/>
                         </if>
                     </if>
+
+                    <!-- trigger the creation of extra searchable taxonomy tags -->
+                    <nhsdigital:searchableTaxonomy variant="draft"/>
```

Comment:
Can we have our custom actions (this one and all the other ones too) renamed so that their names would read as commands, for example: `setSerchableTaxonomy` or `populateSerchableTaxonomy`?

Hippo seems to be following this convention and it does make it more clear that this entry is about executing some activity.

Reply:
👍